packages/editor-ui/src/shims.d.ts

```diff
@@ -17,6 +18,7 @@ declare global {
 	interface Window {
 		BASE_PATH: string;
 		REST_ENDPOINT: string;
+		__n8n_settings: FrontendSettings;
```

Comment:
nit: I feel like `__n8n_settings` is a bit pythonic. I'd personally prefer a plain, clear camelCase name like `n8nSettings`.